docs/ad/index.md

```diff
@@ -40,36 +45,54 @@ A feature is the field in your index that you want to find anomalies for. A dete
 You can add a maximum of five features for a detector.
 {: .note }
 
-1. Select the **Enable feature** check box.
-2. Add the **Feature name**.
-3. For **Feature type**, choose **Field aggregation**, and then choose the aggregation method. Or choose **Custom expression**, and add in your own JSON aggregation query.
-4. Choose a **Field**.
-5. Choose **Save**.
+1. On the **Features** page, select **Add features**.
+1. Enter the **Name** of the feature.
+1. For **Find anomalies based on**, choose the method to find anomalies. For **Field Value** menu, choose the **field** and the **aggregation method**. Or choose **Custom expression**, and add in your own JSON aggregation query.
+1. Preview sample anomalies and adjust the feature settings if needed.
+- For sample previews, the anomaly detection plugin selects a small number of data samples, for example, one data point for every 30 minutes, and uses interpolation to estimate the remaining data points to approximates the actual feature data. It loads this sample dataset into the detector. The detector uses this sample dataset to generate a sample preview of anomaly results.
+Examine the sample preview and use it to fine-tune your feature configurations, for example, enable or disable features, to get more accurate results.
+1. Choose **Save**.
```

Comment:
We changed button name as "Save and start detector" based on UX review comments

Reply:
Done.